packages/transform-to-vanilla/src/transform-keys/merge-key.ts

```diff
@@ -0,0 +1,26 @@
+const RegExp = /[$_]$/;
+
+// == Interface ================================================================
+export function removeSignSimbol(keyStr: string) {
+  return hasSignSimbol(keyStr) ? keyStr.trim().replace(RegExp, "") : keyStr;
+}
+
+// == Utils ====================================================================
+function hasSignSimbol(keyStr: string) {
+  return RegExp.test(keyStr);
+}
+
+// == Tests ====================================================================
+if (import.meta.vitest) {
+  const { describe, it, expect } = import.meta.vitest;
+
+  describe.concurrent("Remove Key Sign Symbol", () => {
+    it("No Sign Symbol", () => {
+      expect(removeSignSimbol("boxShadow")).toBe("boxShadow");
+    });
+    it("Has Sign Symbol", () => {
+      expect(removeSignSimbol("boxShadow$")).toBe("boxShadow");
+      expect(removeSignSimbol("transform_")).toBe("transform");
+    });
```

Comment:
Would you add test case for `Ignore Sign Symbol`?
This is because the deletion of letters should only happen at the end.

Example
- `_boxShadow`
- `_transform`
- `box$Shadow`
- `trans_form`

Reply:
Ok i will add more test cases!